src/idrs.jl

```diff
@@ -78,8 +80,8 @@ end
 end
 
 function idrs_method!(log::ConvergenceHistory, X, A, C::T,
-    s::Number, abstol::Real, reltol::Real, maxiter::Number; smoothing::Bool=false, verbose::Bool=false
-    ) where {T}
+    s::Number, Pl::precT, abstol::Real, reltol::Real, maxiter::Number; smoothing::Bool=false, verbose::Bool=false
+    ) where {T, precT}

```

Comment:
I think you never use `precT`, so you don't need that type parameter in the method signature

Reply:
Yes, I just followed the "gmres.jl" and added "precT". Maybe it will be useful in the future?